.deepsource.toml

```diff
@@ -4,7 +4,4 @@ version = 1
 name = "python"
 
   [analyzers.meta]
-  runtime_version = "3.x.x"
-
-[[transformers]]
-name = "black"
```

Comment:
I especially want the **`black`** linter to be used in this project. I believe we can keep these lines.